src/include/detail/linalg/tdb_matrix.h

```diff
@@ -130,7 +130,7 @@ class tdbBlockedMatrix : public MatrixBase {
    */
   tdbBlockedMatrix(const tiledb::Context& ctx, const std::string& uri) noexcept
     requires(std::is_same_v<LayoutPolicy, stdx::layout_left>)
-      : tdbBlockedMatrix(ctx, uri, 0, 0, 0, 0, 0, 0) {
+      : tdbBlockedMatrix(ctx, uri, 0, 0, 0, 0, 0, 0, true) {
```

Comment:
Again, we should not use explicit flag, but just omit all of the view parameters.

And, if there are issues related to ambiguity because of the suggested overloading (I don't think there will be), we should use a class that is a start-stop pair, rather than separate integers.  This is something I have had in mind to do anyway.

Reply:
Sure, thanks, done. Will keep that in mind as well.